docs/how-to-get-swarm.md

```diff
@@ -0,0 +1,191 @@
+        .
+        .
+        .
+        f6dc2febe7d321b987d30228e8c7a21b
+
+    The last line of the output is the discovery token.
+
+3. Copy the discovery token to a safe place.
+
+### Create the Swarm nodes individually
+
+You can use this procedure to deploy a small Swarm cluster with a somewhat static number of nodes. To deploy larger clusters, or to dynamically scale the size of a cluster, consider using Swarm management software, such as Orchestrate. <For more information, refer to the "Choosing Swarm management software" topic in the Additional resources subsection>   
+
+1. Connect your Docker client software to the Docker Engine where you plan to run the primary Swarm manager.<For example: (without using docker-machine)>
+
+2. Use the following syntax to run a Swarm container as the primary Swarm manager.
+
+        docker run -t -p <your_selected_port>:2375 -t swarm manage token://<cluster_id> >
+
+    For example:
+
+        docker run -t -p 2370:2375 -t swarm manage token://f6dc2febe7d321b987d30228e8c7a21b
+
+3. Connect the Docker client software to the Docker Engine where you plan to run a Swarm agent.
+
+4. Use the following syntax to run a Swarm container as an agent.
+
+        docker run -d swarm join --addr=<node_ip:2375> token://<cluster_id>
+
+    For example:
+
+        docker run -d swarm join --addr=192.168.99.101:2375 token://f6dc2febe7d321b987d30228e8c7a21b
+
+5. Repeat the previous two steps for each node where you plan to run a Swarm agent.
+
+### Manage your Swarm
+
+1. Connect the Docker Client to the Swarm manager.
+
+2. Verify that all of the Swarm nodes appear in the output from the following command.
+
+        $ docker info
+
+3. Check the images currently running on your Swarm cluster.
+
+        $ docker ps
+
+4. Run a container on the Swarm cluster. For example:
+
+        $ docker run hello-world
+        Hello from Docker.
+        .
+        .
+        .
+
+5. Use `docker ps` to find out which node the container ran on.
+
+        $ docker ps  -a
+
+
+## Run a Swarm binary
+
+<Rolfe - Insert diagram of binary on host computer.>
+
+Before you run a Swarm binary directly on a host operating system (OS), you compile the binary from the source code or get a trusted copy from another location. Then you run the Swarm binary.
+
+Using a Swarm binary this way has one key benefit over other methods: If you are a developer who contributes to the Swarm project, you can test your code changes without "containerizing" the binary before you run it.
+
+Running a Swarm binary on the host OS has disadvantages: Getting and running the latest version of Swarm is a burden. The binary doesn't get the benefits that Docker containers provide, such as isolation. Most documentation and tutorials aren't based on this method of running Swarm. Lastly, because the swarm nodes don't use Docker Engine, you can't use Docker-based software tools, such as Docker Client an Orchestrate at the node level.   
+
```

Comment:
I'll fix these issues in the updated version. After I fact-check it, the updated version will say something like: "Lastly, because these swarm binaries don't use Docker Engine, you can't use Docker orchestration tools such as Docker Machine and Docker Compose to manage them."